src/status_im/contexts/syncing/device/view.cljs

```diff
@@ -1,17 +1,47 @@
 (ns status-im.contexts.syncing.device.view
   (:require
     [quo.core :as quo]
+    [react-native.core :as rn]
     [status-im.contexts.syncing.device.style :as style]
-    [utils.i18n :as i18n]))
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- hide-sheet
+  []
+  (rf/dispatch [:hide-bottom-sheet]))
+
+(defn- unpair-confirmation
+  [installation-id name]
+  (fn []
+    (let [unpair-device #(rf/dispatch [:syncing/disable-installation installation-id])]
+      [rn/view
+       [quo/drawer-top
+        {:title       (i18n/label :t/unpair-device)
+         :description (i18n/label :t/unpair-device-description {:name name})}]
+       [quo/bottom-actions
+        {:actions          :two-actions
+         :button-one-label (i18n/label :t/unpair)
+         :button-one-props {:type     :danger
+                            :on-press unpair-device}
+         :button-two-label (i18n/label :t/cancel)
+         :button-two-props {:type     :grey
+                            :on-press hide-sheet}}]])))
 
 (defn view
   [{:keys [name
+           installation-id
            this-device?
            device-type
            enabled?
            show-button?]}]
-  (let [paired?   (and (not this-device?) enabled?)
-        unpaired? (not enabled?)]
+  (let [paired?                  (and (not this-device?) enabled?)
+        unpaired?                (not enabled?)
+        show-unpair-confirmation (fn []
+                                   (rf/dispatch [:show-bottom-sheet
+                                                 {:theme   :dark
+                                                  :content (unpair-confirmation installation-id name)}]))
```

Comment:
If this is hiccup, Shouldn't it be rendered using brackers `[]` instead of parentheses `()`?

Reply:
It's a function that returns hiccup